config.json

```diff
@@ -1,6 +1,8 @@
 {
   "language": "Haskell",
   "active": true,
+  "solution_pattern": "example.*[.]hs",
+  "ignore_pattern": "[Ee]xample",
```

Comment:
Trackler is about to be retired. That said, if this track always uses `example` and never `Example` then I think we should simplify to what is actually used.